### DISCUSSION TITLE
DynamoDB Pagination in Node js

### DISCUSSION THREAD

**Discussion by mallikarjuna11421** [aws/aws-sdk-js#4146] [General]
Hi 

I am facing issue in Node JS with DynamoDB.

I am trying to get the data from table using Query operation with Limit 100;

for 78 records it reached to 1 MB. how can i get next records from 79 to 100 and LastEvaluatedKey.

My Code:
-------
let params = {
            TableName: "TableName",
            IndexName: 'SearchByName',
            Limit: 100,
            KeyConditionExpression: "channel = :type",
            ProjectionExpression: "version,channel",
            ExpressionAttributeValues: {
                ":type": search
            }
        };

        let dbresponse=await docClient.query(params).promise();

Can you guys help me to resolve this.

**Comment by BohdanPetryshyn** [comment]
@mallikarjuna11421, you can pass the `LastEvaluatedKey` from the response to the next `docClient.query()` call as `ExclusiveStartKey` parameter

**Reply by mallikarjuna11421** [reply]
@BohdanPetryshyn  Thanks for reply.
Yes i tried what you suggested but i am getting next 100 records data .so total i am getting 175 records.But the requirement here is i need only 100 records.
